src/helpers/script_helpers.sh

```diff
@@ -122,3 +145,42 @@ function run_steps {
     remove_step_files
   fi
 }
+
+
+# Skip any steps on the current branch
+function skip_current_branch_steps {
+  local file="$1"
+  local add_noop="$2"
+
+  while [ "$(has_lines "$file")" = true ]; do
+    if [[ "$(peek_line "$file")" =~ ^checkout ]]; then
+      if [ -n "$add_noop" ]; then
+        prepend_to_file "noop" "$file"
+      fi
+
```

Comment:
I don't think this empty line adds readability. I find it rather confusing, and would vote to remove it.